src/App.js

```diff
@@ -1,12 +1,31 @@
 import { useEffect, useState } from "react";
+import WeatherByHourData from "./components/WeatherByHourData";
 import './App.css';
 import logo from './mlh-prep.png'
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import { addDays } from "date-fns";
+import calendarIcon from './images/55281.png'
 
-function App() {
+function App(props) {
   const [error, setError] = useState(null);
   const [isLoaded, setIsLoaded] = useState(false);
   const [city, setCity] = useState("New York City")
   const [results, setResults] = useState(null);
+  const [date, setDate] = useState(new Date());
+  const arrayOfTimes = ["SelectTime", "00:00:00", "03:00:00", "06:00:00","09:00:00","12:00:00", "15:00:00", "18:00:00","21:00:00"]
+  const [timeOption, setTimeOption] = useState(arrayOfTimes[0]);
+
+
+  const ExampleCustomTimeInput = () => (
```

Comment:
Not a major issue in this case but it's generally not good to create components within other components since they will be re-created every time the parent component renders.